### PR TITLE
[BUG][Alert]fix alert plugin DingTalkSender

### PR DESCRIPTION
**Pull request by felix-thinkingdata**
fix alert plugin DingTalkSender

image

### CODE REVIEW

File: dolphinscheduler-alert-plugin/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkAlertChannelFactory.java [hunk 1]
```diff
@@ -45,11 +45,6 @@ public String getName() {
                 .setRequired(true)
                 .build())
             .build();
-        InputParam keywordParam = InputParam.newBuilder(DingTalkParamsConstants.NAME_DING_TALK_KEYWORD, DingTalkParamsConstants.DING_TALK_KEYWORD)
```
Comment (CalvinKirs) [inline dolphinscheduler-alert-plugin/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkAlertChannelFactory.java:48]: If you delete this field, the message may not be sent out.

Comment (felix-thinkingdata) [reply dolphinscheduler-alert-plugin/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkAlertChannelFactory.java:48]: Is testing，Wait me the result

Comment (felix-thinkingdata) [reply dolphinscheduler-alert-plugin/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkAlertChannelFactory.java]: image
i see

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**